src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidatorTest.java

```diff
@@ -506,12 +541,116 @@ void testErrorRegisteredInCountryFormedInNotSupplied() {
     }
 
     @Test
-    void testErrorRegisteredInCountryFormedInSupplied() {
+    void testNoErrorRegisteredInCountryFormedInSupplied() {
         trustDataDtoList.get(0).getCorporates().get(0).setOnRegisterInCountryFormedIn(Boolean.TRUE);
-        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testNoErrorIsStillInvolvedAndCeasedDateNull() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateBodyStillInvolvedInTrust(Boolean.TRUE);
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(null);
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testNoErrorIsNotStillInvolvedAndCeasedDateNotNull() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateBodyStillInvolvedInTrust(Boolean.FALSE);
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(LocalDate.of(2020, 1, 1));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testErrorIsStillInvolvedNull() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateBodyStillInvolvedInTrust(null);
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD, TrustCorporateDto.IS_CORPORATE_BODY_STILL_INVOLVED_IN_TRUST_FIELD);
+        String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorIsStillInvolvedAndCeasedDateNotNull() {
```

Comment:
i's change test name to testErrorIsStillInvolvedTrueAndCeasedDateNotNull as IsStillInvolvedAndCeasedDateNotNull reads as if both are not null, could be false but testing for true

Reply:
Updated all new tests to follow this naming pattern, did see comment below, if we change this should change all.

Reply:
Looks like this is addressed. Will resolve.